backend/tests/conftest.py

```diff
@@ -167,6 +168,14 @@ async def default_ipnamespace(db: InfrahubDatabase, register_core_models_schema)
     return None
 
 
+@pytest.fixture
+def default_permission_backend() -> Generator[None, Any, Any]:
+    if not registry.permission_backends:
+        registry.permission_backends = [LocalPermissionBackend()]
+    yield
+    registry.permission_backends = []
```

Comment:
What is it we want to do with this fixture? To me this look wrong in the sense that if another permissions_backend is set it will not change it back to the default and then once done it will remove all permissions backends.

Don't we want a copy to begin with and then assign `[LocalPermissionBackend()]` followed by a yield and then we revert to the original value of the permissions_backend?